delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/Resources.scala

```diff
@@ -382,6 +403,16 @@ object Resources {
       // format: on
     }
 
+    def updateResourceSchema(u: UpdateResourceSchema) = {
+      for {
+        s                          <- stateWhereResourceIsEditable(u)
+        schemaRef                  <- IO.fromOption(u.schemaOpt)(???)
```

Comment:
???

Reply:
Ah, this option will actually never be empty, it's just an option because that's what `ModifyCommand` expects. Weird it slipped through the cracks

Reply:
I tried to address it gracefully, cf. new commits